testutil/compose/static/lodestar/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# TODO(dhruv): use official release version tag.
+FROM chainsafe/lodestar:next
```

Comment:
[v1.8.0-rc.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.8.0-rc.0) is already released, we should have a stable release in ~2 days if there are no issues during rc testing phase